src/xrPhysics/xrPhysics.cpp

```diff
@@ -13,6 +13,16 @@ static void* ode_alloc(size_t size) { return xr_malloc(size); }
 static void* ode_realloc(void* ptr, size_t oldsize, size_t newsize) { return xr_realloc(ptr, newsize); }
 static void ode_free(void* ptr, size_t size) { return xr_free(ptr); }
 
+#ifdef LINUX
+__attribute__((constructor))
+#endif
+static void load(int argc, char** argv, char** envp)
```

Comment:
const char** ?

Reply:
But looks like we don't expect the modification of the arguments. So, making the arguments 'const' should help us to avoid occasional modifying data in the future. It makes our method safer. In addition, the 'const' variables allow the compiler to use advanced optimization techniques sometimes.